examples/go/cat.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2019, 2020 Aleksa Sarai <cyphar@cyphar.com>
+// Copyright (C) 2019, 2020 SUSE LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"../../contrib/bindings/go/pathrs"
```

Comment:
I would really prefer `github.com/openSUSE/libpathrs/contrib/bindings/go` or whatever. To be honest I wasn't even aware you could do this...

Reply:
To be candor I am now sure that it is possible with
https://github.com/openSUSE/libpathrs/blob/feceead820ac9252932bff28872db6cfbc092a0b/contrib/bindings/go/pathrs/pathrs.go#L19-L21

Reply:
`LDFLAGS` are separate to Go import paths. If you clone this repo (`libpathrs`) into `$GOPATH/src/github.com/openSUSE/libpathrs` then the import `github.com/openSUSE/libpathrs/contrib/bindings/go` will work.

However, you're right that this is a pain to import. I will think about this some more, so you can just leave it as-is...